utils/validators/setting.js

```diff
@@ -19,7 +19,7 @@ export function backupTarget(value, getters, errors, validatorArgs) {
   const type = parseValue.type;
 
   if (!type) {
-    errors.push(t('validation.required', { key: 'Type' }));
+    return true;
```

Comment:
I'm not 100% sure, but this be `return errors;` (inline with all other validators)?

Reply:
Done.